card/lib/generators/card/templates/create_cards_tables.erb

```diff
@@ -0,0 +1,106 @@
+# encoding: UTF-8
```

Comment:
If you want to re-use one source of the schema, I think schema.rb should be derived from a migration like this, not the other way around.  After all, in general an app starts without a schema and migrations build it up.

I'm not totally attached to the prefix concept, and in general maybe we encourage people not to use it, but it seems a necessary option for deck apps.  What if they already have a table named 'card', for example?  We probably don't need to support for the app deck case, and no doubt there are a lot of fixes needed to actually support it.

Is there any reason for adding a MyDeck::Engine?  You could create one, but in general I don't think it is needed.  Is there something wrong with doing Decko::Engine.config.foo_bar = 'baz' ?

I'm just at the point of trying to test generating a new rails app, adding the card engine and trying to run it.  I was planning on adding a script like the one for the migration test as a developer guideline.

Don't you agree that